src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php

```diff
@@ -312,6 +313,8 @@ private function convertParameters(array $parameters, string $type, \DOMElement
                 $element->setAttribute('type', 'binary');
                 $text = $this->document->createTextNode(self::phpToXml(base64_encode($value)));
                 $element->appendChild($text);
+            } elseif ($value instanceof AbstractArgument) {
+                throw new RuntimeException($value);
```

Comment:
it should be possible to dump a container in xml even if it contains an abstract argument
this should be dumped with `type="abstract"`

Reply:
fixed